vars/runTestFunctionalV1.groovy

```diff
@@ -19,6 +19,7 @@ def call(Map config = [:]) {
    * config['pragma_suffix'] The Test-tag pragma suffix
    * config['test_tag'] The test tags to run
    * config['ftest_arg'] An argument to ftest.sh
+   * config['test_repeat'] Number of times to repeat a functional test
```

Comment:
I wouldn't be sad if this version of this function didn't get this feature.  It's historical at this point.  Although, now that you have done the work…

That said, there is no mention of a default value for this parameter.  It should have a default value.

Reply:
Where are the mentions of the default value for the other parameters, e.g. `ftest_arg`?

Reply:
`ftest_arg` defaults to empty.  But you are right, that perhaps it's useful to have that documented.  We do document default values elsewhere but obviously have not been consistent.

My question though was more about whether there was a reasonable default value for that parameter (from the top of the call-stack down to `launch.py`) and what is that default value(?), more than just documenting it.  It seems that having it default to 1 seems a reasonable default, but I'm not seeing where that happens and it is being passed to ftest as a mandatory argument (i.e. in quotes).  Is `ftest.sh` turning an empty value there into a default of 1 (yes, I discovered)?  That said, I don't think I like adding another mandatory (but with reasonable default) argument to `ftest.sh` either which I have commented in more detail about in daos-stack/daos#5678.

None of this is a show-stopper that must be resolved here.  I just wanted to make my case and get your thoughts about it.  The comments in daos-stack/daos#5678 explain it in more detail.

Reply:
I've revised my edits to no longer use an additional `test_repeat` config parameter.  Instead all optional arguments to launch.py - passed through `ftest.sh` - are defined through `config['ftest_arg']`.  As a result any `launch.py` argument must be fully defined in `config['ftest_arg']`, so passing the optional `--nvme` argument is done by assigning `config['ftest_arg']="--nvme auto:Optane"` instead of `config['ftest_arg']="auto:Optane"`.  When also including the `--repeat` argument, through the `Test-repeat*` pragma, this assignment will look like `config['ftest_arg']="auto:Optane --repeat 2"`.